sdk/schemaregistry/azure-schemaregistry-avroserializer/README.md

```diff
@@ -27,7 +27,22 @@ To use this package, you must have:
 * Python 2.7, 3.6 or later - [Install Python][python]
 
 ### Authenticate the client
```

Comment:
I think maybe we could drop this section?
since for Avro Serializer, we just pass in the SRClient.

What about other languages' readme for the serializer?

Reply:
.NET does have this section, but they specify that the constructor to pass things into is the client constructor: https://github.com/Azure/azure-sdk-for-net/tree/main/sdk/schemaregistry/Microsoft.Azure.Data.SchemaRegistry.ApacheAvro#authenticate-the-client
Java does too: https://github.com/Azure/azure-sdk-for-java/tree/main/sdk/schemaregistry/azure-data-schemaregistry-apacheavro#create-schemaregistryasyncclient-with-azure-active-directory-credential
JS does not have this section.

Personally, I would prefer keeping and re-wording this. I personally as a user would want all necessary info here, and having to click links to authenticate, etc. would not be super convenient.